vispy/color/colormap.py

```diff
@@ -438,6 +439,75 @@ def map(self, t):
                            np.sqrt(t))
 
 
+class _SingleHue(Colormap):
+    """A colormap which is solely defined by the given hue and value.
+
+    Given the color hue and value, this color map increases the saturation
+    of a color. The start color is almost white but still contains a hint of
+    the given color, and at the end the color is fully saturated.
+
+    Parameters
+    ----------
+    hue : scalar, optional
+        The number indicating the hue value. Must be in the range [0, 360].
+        Defaults to 200 (blue).
+    value : scalar, optional
+        The number represting the value component of a color. Must be in the
+        range [0, 1.0]. The default value is 1.0.
+    """
+
+    def __init__(self, hue=200, value=1.0):
+        colors = ColorArray([
+            (hue, 0.05, value),
+            (hue, 1.0, value)
+        ], color_space='hsv')
+        super(_SingleHue, self).__init__(colors)
+
+
+class _HSL(Colormap):
+    """A colormap which is defined by n evenly spaced points in
+    a circular color space.
+
+    This means that we change the hue value while keeping the
+    saturation and value constant.
+
+    Parameters
+    ---------
+    n_colors : int (optional)
+        The number of colors to generate.
+    hue_start : int (optional)
+        The hue start value. Must be in the range [0, 360], the default is 0.
+    saturation : float (optional)
+        The saturation component of the colors generated. The default is fully
```

Comment:
The tense is inconsistent. For `n_colors`, you used "to generate", but here it's "generated".